src/web/routes/application/confirm.js

```diff
@@ -4,6 +4,8 @@ const pageContent = {
 }
 
 const getConfirmPage = (req, res) => {
+  req.session.destroy()
```

Comment:
@mikeSmith08 @YetAnotherMatt what are your thoughts on destroying the session on the confirm page? Currently we can hit this page directly so could in theory kill the session before POSTing the form.

Reply:
My opinion on this is that as we don't provide navigation direct to this via any means apart from the user messing with the URL, then its fine that this happens - I'd expect the final page of the flow to so this.